src/android/StatusBar.java

```diff
@@ -142,6 +142,23 @@ public void run() {
             return true;
         }
 
+        if ("overlaysWebView".equals(action)) {
```

Comment:
I am not sure what you mean here. The method has the same name of the equivalent one already existing for the iOS plug-in. Calling the method from JS works fine and has been tested on multiple Android versions.